inline_scan.sh

```diff
@@ -292,8 +293,6 @@ prepare_inline_container() {
         CREATE_CMD+=('-e VERBOSE=true')
         RUN_CMD+=('-e VERBOSE=true')
     fi
-    printf '\n%s\n' "Creating volume mount -- ${VOLUME_PATH}:/anchore-engine"
```

Comment:
Completely avoid to use volumes, since we can't be sure they'll work (eg: docker-in-docker).
Proceeding always with `docker cp`